climatetree/places-microservice
# Merge to Master

### Change type:
- New features


### Description:
- Returns geoJson objects from both API endpoints
- Parameterized getSimilarPlaces API
- Adds nation, state, and county names to the returned geoJson properties list

### Trello Card Link:
https://trello.com/c/tIkwTIT0/51-update-similar-places-api-to-accept-arguments-for-population-percentage-range-and-join-on-nation-state-and-county-info

https://trello.com/c/xblMuJsu/52-update-find-place-by-name-api-to-join-place-info-on-nation-state-and-county

https://trello.com/c/qmeRR8QA/47-update-url-for-place-search-with-name

https://trello.com/c/Pj73qkf0/48-update-places-microservice-api-endpoint-to-update-sending-geojson-objects-as-response-for-querying-for-a-database

### PR Checklist:

### Steps to Verify Pull Request
- Start spring boot app
- Hit api /places/Manchester
- Hit api/places/3233/similar
    - Should return all states within 95%-150% population (default percents) of Massachussetts
- Hit api/places/3233/similar?populationStart=80&populationEnd=110
    - Should return all states within 80%-110% population of Massachussetts
- Verify response in http://geojsonlint.com/
